data/WI/incentives.json

```diff
@@ -0,0 +1,410 @@
+      "number": 50,
+      "minimum": 50,
+      "maximum": 50
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$50 for Smart Thermostats. Available as instant discounts in the online marketplace or as rebates."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "bonus_available": true,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-8",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "wi_focusOnEnergy_waterHeatingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 300,
+      "minimum": 300
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Instant discounts for electric heat pump water heaters starting at $300 through a Trade Ally contractor."
+    },
+    "bonus_available": true,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-10",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "rooftop_solar_installation",
+    "program": "wi_focusOnEnergy_residentialCustomerSolarPVRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500,
+      "minimum": 500,
+      "maximum": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rural residential customers can receive an additional $500 for installing a system."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-11",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "wi_focusOnEnergy_dIYResidentialInsulation&AirSealingRebate",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 200,
+      "minimum": 200,
+      "maximum": 200
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$200 cash back for qualified self-installed attic insulation and air sealing."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-21",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "wi_focusOnEnergy_residentialHeating&CoolingRebate",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1300,
+      "minimum": 1300,
+      "maximum": 1300
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$1,300 for Cold Climate Air Source Heat Pump (existing primary heating fuel: natural gas or electric)."
```

Comment:
For all of these, this phrasing seems a bit more succinct: $1,300 for Cold Climate Air Source Heat Pump replacing natural gas or electric primary heating source.